waiter/config-full.edn

```diff
@@ -263,6 +263,49 @@
                                ;; The URL for your Marathon HTTP API:
                                :url "http://marathon.example.com:8080"}
 
+                    ;; :kind :kubernetes uses Kubernetes (https://kubernetes.io/) for scheduling Waiter services and instances:
+                    ;:kind :kubernetes
+                    :kubernetes {
+                                 ;; Configuration for obtaining the credentials to authenticate with the Kubernetes API server.
+                                 ;; If authentication isn't required (e.g., using kubectl proxy), then this can be omitted.
+                                 :authentication {;; Number of minutes to wait between attempts to refresh the credentials.
+                                                  ;; If omitted, then the :refresh-fn value is only called once at Waiter startup.
+                                                  :refresh-delay-minutes 15
+
+                                                  ;; Nullary function which can be called periodically (or just once if :refresh-delay-minutes is omitted)
+                                                  ;; to obtain an Authorization string for use when authenticating with the Kubernetes API server.
+                                                  ;; The function must return a string that can be used as the value for the Authorization HTTP header.
+                                                  ;; The sample function given below simply returns the string from the WAITER_K8S_AUTH_STRING environment variable.
+                                                  :refresh-fn waiter.scheduler.kubernetes/authorization-from-environment}
+
+                                 ;; HTTP options that will be used when accessing the Kubernetes API:
+                                 :http-options {:conn-timeout 10000
+                                                :socket-timeout 10000}
+
+                                 ;; Number of retry attempts to make when a PATCH request fails with a 409 Conflict:
+                                 :max-conflict-retries 5
+
+                                 ;; Maximum number of characters allowed for a Kubernetes object name:
+                                 :max-name-length 63
+
+                                 ;; String value used to annotate Kubernetes objects that are orchestrated by this Waiter instantiation:
+                                 :orchestrator-name "waiter"
+
+                                 ;; Port number used to set the $PORT0 (service port) for Waiter service instances.
+                                 ;; If more than one port is requested, $PORT1 thru $PORT9 will be defined sequentially after $PORT0
+                                 ;; (e.g., if pod-base-port is 8080, then $PORT0==8080 and $PORT9==8089):
+                                 :pod-base-port 31000
```

Comment:
Ports should be "random"ized.  I'm thinking something based upon the `service-id`, confined within a certain range.